app/Filament/Resources/InterpretationResource.php

```diff
@@ -49,8 +49,22 @@ public static function table(Table $table): Table
         return $table
             ->columns([
                 Tables\Columns\TextColumn::make('name'),
-                Tables\Columns\TextColumn::make('namespace'),
-                Tables\Columns\TextColumn::make('route'),
+                Tables\Columns\TextColumn::make('context')
+                    ->label(__('Show context'))
+                    ->getStateUsing(fn (Interpretation $record): string => __('Show context').' <span class="sr-only">'.__('for').$record->name.'</span>')
```

Comment:
Missing a space between "for" and the name.

```suggestion
                    ->getStateUsing(fn (Interpretation $record): string => __('Show context').' <span class="sr-only">'.__('for :name', ['name' => $record->name]).'</span>')
```

Reply:
Is it possible to add this extra info to the "Edit" button too?